src/utils/addDependenciesToPackages.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import Project from '../Project';
+import type Workspace from '../Workspace';
+import type Package from '../Package';
+import type { Dependency, configDependencyType } from '../types';
+import * as logger from './logger';
+import * as yarn from './yarn';
+
+export default async function addDependenciesToPackage(
+  project: Project,
+  pkg: Package,
+  dependencies: Array<Dependency>,
+  type?: configDependencyType
+) {
+  const isProjectPackage = project.pkg.dir === pkg.dir;
```

Comment:
```js
pkg.isSamePackage(project.pkg)
```

Reply:
Ah yea,  I was waiting for the remove PR.